patches/components-minidump_uploader-android-java-src-org-chromium-components-minidump_uploader-MinidumpUploader.java.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
+index 645a041d9ac32120178a1988b90a4fb3d2ab450a..8835a49169e19d1cf328cc22e401b8e571ddac77 100644
+--- a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
++++ b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/MinidumpUploader.java
+@@ -26,7 +26,7 @@ import java.util.zip.GZIPOutputStream;
+  */
+ public class MinidumpUploader {
+     /* package */
+-    static final String CRASH_URL_STRING = "https://clients2.google.com/cr/report";
++    static final String CRASH_URL_STRING = "https://cr.brave.com";
```

Comment:
I tried to make it with bytecode asm, but it didn't work.
My failed attempt is here https://github.com/brave/brave-core/commit/d0a390984f697ea018f87f83c2476640b1fe4750 .
If there is a way how to make that without patch, I will be happy to re-do.

Reply:
Does your attempt generate build errors?
I suppose we can add `changeFieldOwner` similar to `changeMethodOwner`, maybe as a follow up as it is not related to this task directly.

Reply:
No, my best attempt did not generate a build error, just the actual url where crash dump was sent was the one from Google, actual url substitution didn't happen. And besides all I had to add two more new patches.